dts/arm/nordic/nrf52840.dtsi

```diff
@@ -174,6 +174,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
+			compatible = "nordic,nrf-cc310";
+			reg = <0x5002A000 0x1000>;
+			label = "CRYPTOCELL";
+			status = "ok";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			cryptocell310: crypto@5002b000 {
+				compatible = "arm,cryptocell-310";
+				reg = <0x5002B000 0x1000>;
+				interrupts = <42 1>;
+				interrupt-names = "cc310";
```

Comment:
we should probable remove interrupt-names if its just 'cc310', doesn't have much meaning.  interrupt-names is usually useful when we have multiple interrupts from a device, like a uart with one interrupt for rx & one for tx. The name should represent what the interrupt is conveying (completion, error, etc).